homeassistant/components/ld2410_ble/binary_sensor.py

```diff
@@ -0,0 +1,71 @@
+"""LD2410 BLE integration light platform."""
+
+
+from homeassistant.components.binary_sensor import (
+    BinarySensorDeviceClass,
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+)
+
+from . import LD2410BLE
+from .const import DOMAIN
+from .models import LD2410BLEData
+
+ENTITY_DESCRIPTIONS = {
+    "is_moving": BinarySensorEntityDescription(
+        key="is_moving", device_class=BinarySensorDeviceClass.MOTION
+    ),
+    "is_static": BinarySensorEntityDescription(
+        key="is_static", device_class=BinarySensorDeviceClass.PRESENCE
+    ),
+}
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the platform for LD2410BLE."""
+    data: LD2410BLEData = hass.data[DOMAIN][entry.entry_id]
+    async_add_entities(
+        [
```

Comment:
You can drop the `[]` as well as a generator expression works here

Reply:
Done.